torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -1204,9 +1204,12 @@ at::Tensor& AtenXlaType::elu_(at::Tensor& self, at::Scalar alpha,
 
 at::Tensor AtenXlaType::elu_backward(const at::Tensor& grad_output,
                                      at::Scalar alpha, at::Scalar scale,
-                                     at::Scalar input_scale,
+                                     at::Scalar input_scale, bool self_or_result,
```

Comment:
I had meant `output` should be named to `self_or_result` and the bool param should remain `self`